sdcm/tester.py

```diff
@@ -162,7 +162,7 @@ def setUp(self):
         self.db_cluster.wait_for_init()
         db_node_address = self.db_cluster.nodes[0].private_ip_address
         self.loaders.wait_for_init(db_node_address=db_node_address)
-        nodes_monitored = [node.public_ip_address for node in self.db_cluster.nodes]
+        nodes_monitored = [node.private_ip_address for node in self.db_cluster.nodes]
```

Comment:
Ah, sure, I haven't thought about that. Don't we need to update that in other parts of the code (some tests do some custom setup, so it's better to look into them and change the references).